### PR TITLE
Fix android login infinite loading (case 18459)

### PR DESCRIPTION
**Pull request by gcalero**
Login fixes and improvement

**Manuscript cases:**
18459 - Android: Putting incorrect username and password may create infinite loading screen
18388 - Android: Switching to another app while on log in page will yield infinite loading screen

**Improvement:**
+ Adding the ability to show password

**Test plan**
- Using the right username and password must login the user
- Using the right email and password must login the user
- Using wrong username/email or password must not login the user
- Login should be robust and must conclude following the steps described in both manuscript cases (18459, 18388)
- The password must be shown or hidden pressing the eye at the right of the EditText

Notes:
Username is not case-sensitive, so varying case in the username should work anyways

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13049/HighFidelity-Beta-PR14013-78b4139.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13049/HighFidelity-Beta-PR14013-78b4139.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13049/HighFidelity-Beta-Interface-PR14013-78b4139.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13049/HighFidelity-Beta-PR14013-78b4139.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13049/HighFidelity-Beta-Interface-PR14013-78b4139.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13081/HighFidelity-Beta-PR14013-32da2cb.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13081/HighFidelity-Beta-Interface-PR14013-32da2cb.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13081/HighFidelity-Beta-PR14013-32da2cb.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13081/HighFidelity-Beta-Interface-PR14013-32da2cb.exe

**Comment by qliacoombs** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13212/HighFidelity-Beta-PR14013-32da2cb.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13212/HighFidelity-Beta-Interface-PR14013-32da2cb.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13212/HighFidelity-Beta-PR14013-32da2cb.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13212/HighFidelity-Beta-Interface-PR14013-32da2cb.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13245/HighFidelity-Beta-PR14013-32da2cb.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13245/HighFidelity-Beta-PR14013-32da2cb.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13245/HighFidelity-Beta-Interface-PR14013-32da2cb.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13245/HighFidelity-Beta-PR14013-32da2cb.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13245/HighFidelity-Beta-Interface-PR14013-32da2cb.dmg

**Comment by QliCknox** [comment]
PASS @conklin94122 

Logging in on Android works as expected

File: android/app/src/main/java/io/highfidelity/hifiinterface/fragment/LoginFragment.java [hunk 1]
```diff
@@ -125,10 +91,19 @@ public void onDetach() {
         mListener = null;
     }
 
+    @Override
+    public void onResume() {
+        super.onResume();
+        // This hack intends to keep Qt threads running even after the app comes from background
```
Comment (SamGondelman) [inline android/app/src/main/java/io/highfidelity/hifiinterface/fragment/LoginFragment.java]: Can you explain this?

**Pull request merged**